pkg/controller/nodes/executor.go

```diff
@@ -21,6 +21,8 @@ import (
 	"fmt"
 	"time"
 
+	events2 "github.com/flyteorg/flytepropeller/pkg/controller/events"
```

Comment:
why events2?

Reply:
we import events from flyteidl https://github.com/flyteorg/flytepropeller/pull/304/files/8596eec61953744c10f0093ccea79630cd292af4#diff-8d8b391e178cdede077b3d0cc2e81e1fd8f0754a6c3ffc23be253c8b247a45a1R35 when we initialize the node and task recorders in NewExecutor

Reply:
maybe `idlEvents` and `ctrlEvents` or something :D ?